apps/private-law/prl-cron-fm5-reminder/demo-image-policy.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
 metadata:
   name: demo-prl-cron-fm5-reminder
-  annotations:
-    hmcts.github.com/prod-automated: disabled
```

Comment:
If you are deploying image prl-cos, you dont need to make changes to this file

Reply:
Yes, we don't need these config as FM5 changes are merged to master already.

Reply:
This is just to revert config added to deploy PR image